frontend/src/scenes/insights/InsightTabs/FunnelTab/ToggleButtonChartFilter.tsx

```diff
@@ -15,36 +16,38 @@ export function ToggleButtonChartFilter({
     onChange = noop,
     disabled = false,
 }: ToggleButtonChartFilterProps): JSX.Element {
-    const { chartFilter } = useValues(chartFilterLogic)
-    const { setChartFilter } = useActions(chartFilterLogic)
-    const defaultDisplay = ChartDisplayType.FunnelViz
+    const logic = chartFilterLogic({ defaultChartFilter: ChartDisplayType.FunnelViz })
+    const { chartFilter } = useValues(logic)
+    const { setChartFilter } = useActions(logic)
 
-    const options = [
-        {
-            value: ChartDisplayType.FunnelViz,
-            label: 'Funnel conversion',
-        },
-        {
-            value: ChartDisplayType.ActionsLineGraphLinear,
-            label: 'Conversion trend',
-        },
-    ]
+    const options: { [key in ChartDisplayType]?: string } = {
+        [ChartDisplayType.FunnelViz]: 'Funnel conversion',
+        [ChartDisplayType.ActionsHistogramChart]: 'Time to convert',
+        [ChartDisplayType.ActionsLineGraphLinear]: 'Conversion trend',
+    }
 
     return (
-        <Radio.Group
-            key="2"
-            defaultValue={defaultDisplay}
-            value={chartFilter || defaultDisplay}
-            onChange={({ target: { value } }: { target: { value?: ChartDisplayType } }) => {
-                if (value) {
-                    setChartFilter(value)
-                    onChange(value)
-                }
-            }}
+        <Dropdown
```

Comment:
While visually this does look cleaner, and better overall, I'm not so sure about this. The problem I see is we're losing some discoverability on these features. CC @clarkus for thoughts too.

Reply:
@paolodamico I see your point on discoverability. Since there are only three items, I could see showing them all in an inline list to improve a user's ability to discover them. Especially conversion trends... that's something we're trying to learn more about, so being last in the menu doesn't help with that goal.